common/ballot.go

```diff
@@ -0,0 +1,12 @@
+package common
+
+type Ballot struct {
```

Comment:
I think ballot type can be defined in `vote.go`, it has not use outside of voting.